app/src/main/java/fr/free/nrw/commons/upload/UploadController.java

```diff
@@ -93,7 +93,7 @@ public void startUpload(final Contribution contribution, final ContributionUploa
             contribution.setDescription("");
         }
 
-        String license = prefs.getString(Prefs.DEFAULT_LICENSE, Prefs.Licenses.CC_BY_SA_3);
+        String license = prefs.getString(Prefs.DEFAULT_LICENSE, null);
```

Comment:
`license` is nullable now. Add null checks before using it anywhere. Its used below in 

```
Utils.licenseNameFor(license)
```